scripts/publish.ps1

```diff
@@ -0,0 +1,74 @@
+$ErrorActionPreference = "Stop"
+$InformationPreference = "Continue"
+$WarningPreference = "SilentlyContinue"
+
+Trap {
+    Write-Error $_.InvocationInfo.ScriptName -ErrorAction Continue
+    $line = "$($_.InvocationInfo.ScriptLineNumber): $($_.InvocationInfo.Line)"
+    Write-Error $line -ErrorAction Continue
+    Write-Error $_
+}
+
+function CommandAliasFunction {
+    Write-Information ""
+    Write-Information "$args"
+    $cmd, $args = $args
+    & "$cmd" $args
+    if ($LASTEXITCODE) {
+        throw "Exception Occured"
+    }
+    Write-Information ""
+}
+
+Set-Alias -Name ce -Value CommandAliasFunction -Scope script
+
+$AwsAccountId = $(aws sts get-caller-identity --output text --query Account)
+$EcrRegistry = "$AwsAccountId.dkr.ecr.$env:AWS_DEFAULT_REGION.amazonaws.com"
+$Image = "$EcrRegistry/$env:INPUT_ECR_REPOSITORY" + ":" + "$env:IMAGE_VERSION"
+$DockerfilePath = "$env:INPUT_DOCKERFILE_DIR_PATH"
+
+try {
+    $ImageCount = aws ecr list-images --repository-name $env:INPUT_ECR_REPOSITORY | jq '.imageIds | unique_by(.imageDigest) | length'
+    $ImageTags = aws ecr describe-images --repository-name $env:INPUT_ECR_REPOSITORY --max-items $ImageCount --query 'imageDetails[*].imageTags[0]'
+    if ($ImageTags -like "*$env:IMAGE_VERSION*") {
+        Write-Output "Image already pushed. Skipping docker push"
+        $imageRemove = $false
```

Comment:
Would it be possible to update this so that it is more intuitive by avoiding `if (!($imageRemove -eq $false))` in line 70, for example:

*Outside* of the try/catch block, initialize the variable:
```
# this variable name is slightly better because it indicates 'if' an action should be taken
$removeImage = $true  
```

*Inside* the `try`
`$removeImage = $false`

Then in the `finally` block test with 
```
if($true -eq $removeImage) {   # putting the known value first
  Write-Output "Removing image: $($IMAGE)"
  ce docker image rm "$IMAGE"
}
```

Reply:
Done as requested

Reply:
@naveen-variant can you also update this line as suggested? to avoid NOT expression
https://github.com/variant-inc/actions-collection/pull/30/files#diff-2633a4fe34d376cbccd4d6939d19e5885c8809b4170aadf8a439575bc8dc857eR71

Reply:
done